appengine/template.soy

```diff
@@ -74,6 +74,56 @@
   </div>
 {/template}
 
+/**
+ * Top toolbar for page
+ * @param appName Name of application.
+ * @param levelLinkSuffix Any extra parameters for links.
+ * @param hasLinkButton Whether the page has a link button.
+ * @param hasHelpButton Whether the page has a help button.
+ * @param farLeftHtml Additional content to add to farLeft toolbar.
+ */
+{template .headerBar private="true"}
+  <table width="100%">
+    <tr>
+      <td>
+        <h1>
+          {call .titleSpan}
+            {param appName: $appName /}
+          {/call}
+          {if $ij.level}
+            {call .levelLinks}
+              {param level: $ij.level /}
+              {param maxLevel: $ij.maxLevel /}
+              {param lang: $ij.lang /}
+              {param suffix}{if $levelLinkSuffix}{$levelLinkSuffix}{/if}{/param}
+            {/call}
+          {/if}
+        </h1>
+      </td>
+      <td id="header_cta" class="farSide">
```

Comment:
Out of curiosity, what does cta stand for?  California Teachers Association?  Counter Terrorism Agency?

Reply:
Call to action